nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/lossfunctions/impl/LossHinge.java

```diff
@@ -115,6 +118,12 @@ public INDArray computeGradient(INDArray labels, INDArray preOutput, IActivation
                         computeGradient(labels, preOutput, activationFn, mask));
     }
 
+    @Override
+    public SDVariable defineLossArray(@NonNull SameDiff sameDiff, @NonNull SDVariable input,
+            @NonNull SDVariable labels) {
+        return sameDiff.math.max(input.mul(labels).rsub(1), sameDiff.constant(0.0)).sum(true, 1);
```

Comment:
Constant needs datatype (currently double, which might cause problems for float input)